src/config.rs

```diff
@@ -0,0 +1,5 @@
+#[derive(Debug, Clone)]
+pub enum Mode {
+    Repl,
+    OneTime,
```

Comment:
`Batch` would be a better word for this.

Reply:
sound good

Reply:
actually, this enum is not in use anymore, it's a leftover from my previous attempts. will remove